### DISCUSSION TITLE
求助，导入模板时显示错误

### DISCUSSION THREAD

**Discussion by Dreamergoingup** [windingwind/zotero-better-notes#504] [Q&A] [answered]
![无标题](https://github.com/windingwind/zotero-better-notes/assets/132348800/212dfcbb-8ecf-4ac5-b2ff-592837b33ae9)
![无标题1](https://github.com/windingwind/zotero-better-notes/assets/132348800/afab822f-f4f5-4037-a4d9-095402db2e32)
点击第二个选项后，然后选中我自己的一个笔记，点OK，就弹出下面这个错误窗口，求大佬解决！
![无标题3](https://github.com/windingwind/zotero-better-notes/assets/132348800/a79aff43-d001-4318-be0d-2d3f02fa91cb)

**Comment by windingwind** [comment, marked as answer]
不与zotcard模板兼容。请使用`topItem.getField('title')`
